staging/src/k8s.io/apiserver/pkg/cel/library/library_compatibility_test.go

```diff
@@ -45,6 +41,7 @@ func TestLibraryCompatibility(t *testing.T) {
 		"path", "group", "serviceAccount", "resource", "subresource", "namespace", "name", "check", "allowed", "reason",
 		// Kubernetes <1.28>:
 		"errored", "error",
+		"add", "asApproximateFloat", "asInteger", "compareTo", "isGreaterThan", "isInteger", "isLessThan", "isQuantity", "quantity", "sign", "sub",
```

Comment:
this bit is under 1.29, right?

Reply:
do we need to specify which version of the string library is in each Kubernetes version in https://kubernetes.io/docs/reference/using-api/cel/#cel-community-libraries

Reply:
I also don't see any items documented in https://pkg.go.dev/github.com/google/cel-go/ext#Strings as having been introduced in version 2... is that doc out of date?

Reply:
> this bit is under 1.29, right?

Oops, yes, those new functions should be under 1.29. Fixing

> do we need to specify which version of the string library is in each Kubernetes version in https://kubernetes.io/docs/reference/using-api/cel/#cel-community-libraries

I've gone ahead and opened https://github.com/kubernetes/website/pull/42695 to fix this.

> I also don't see any items documented in https://pkg.go.dev/github.com/google/cel-go/ext#Strings as having been introduced in version 2... is that doc out of date?

I'll do a pass and make sure this gets fixed.

Reply:
Re string lib versioning: 

We skipped version 1 which adds format because we were not ready for it.  

Version 2 doesn't add new functions. It [changes](https://github.com/google/cel-go/blob/8a4595524249653b05082a3ce5b1acb75c448844/ext/strings.go#L481) how `join()` handles input that would impact some integrations, but is transparent to k8s.